app/models/player.rb

```diff
@@ -1,4 +1,5 @@
 class Player < ApplicationRecord
   belongs_to :tournament
   has_many :matchups
+  validates :name, :rating, :division, :crosstables_id, :tournament_id, presence: true
```

Comment:
We don't need validations for foreign keys. Those associations are necessary by default